sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -839,6 +839,19 @@ case class AlterTableSetLocationCommand(
 object DDLUtils {
   val HIVE_PROVIDER = "hive"
 
+  val METASTORE_GENERATED_PROPERTIES: Set[String] = Set(
+    "CreateTime",
+    "transient_lastDdlTime",
+    "grantTime",
+    "lastUpdateTime",
+    "last_modified_by",
+    "last_modified_time",
+    "Owner:",
+    "totalNumberFiles",
+    "maxFileSize",
+    "minFileSize"
```

Comment:
This list was initially created for tests, so we didn't carefully check it. Is it possible to get these property names from some constants in the hive lib? like `HiveClientImpl.HiveStatisticsProperties`?

Reply:
In fact, I can't find `Owner` in tblproperties in hive code. We can try `org.apache.hadoop.hive.metastore.api.hive_metastoreConstants`, but this is also not complete. We need to add some properties like `last_modified_by `, `last_modified_time `.

```
public class hive_metastoreConstants {

  public static final String DDL_TIME = "transient_lastDdlTime";

  public static final String HIVE_FILTER_FIELD_OWNER = "hive_filter_field_owner__";

  public static final String HIVE_FILTER_FIELD_PARAMS = "hive_filter_field_params__";

  public static final String HIVE_FILTER_FIELD_LAST_ACCESS = "hive_filter_field_last_access__";

  public static final String IS_ARCHIVED = "is_archived";

  public static final String ORIGINAL_LOCATION = "original_location";

  public static final String IS_IMMUTABLE = "immutable";

  public static final String META_TABLE_COLUMNS = "columns";

  public static final String META_TABLE_COLUMN_TYPES = "columns.types";

  public static final String BUCKET_FIELD_NAME = "bucket_field_name";

  public static final String BUCKET_COUNT = "bucket_count";

  public static final String FIELD_TO_DIMENSION = "field_to_dimension";

  public static final String META_TABLE_NAME = "name";

  public static final String META_TABLE_DB = "db";

  public static final String META_TABLE_LOCATION = "location";

  public static final String META_TABLE_SERDE = "serde";

  public static final String META_TABLE_PARTITION_COLUMNS = "partition_columns";

  public static final String META_TABLE_PARTITION_COLUMN_TYPES = "partition_columns.types";

  public static final String FILE_INPUT_FORMAT = "file.inputformat";

  public static final String FILE_OUTPUT_FORMAT = "file.outputformat";

  public static final String META_TABLE_STORAGE = "storage_handler";

  public static final String TABLE_IS_TRANSACTIONAL = "transactional";

  public static final String TABLE_NO_AUTO_COMPACT = "no_auto_compaction";
}
```

Reply:
> We need to add some properties like last_modified_by , last_modified_time .

It's fine. It's better to do less hardcode.

Reply:
OK.  
But we can't use hive constants in sql module because of dependency, it seems like hardcode is necessary. Is it fine ?

Reply:
can we remove these properties in `HiveClientImpl.convertHiveTableToCatalogTable`?

Reply:
I think no. 
This behavior is only at `CreateTableLikeCommand`.  `HiveClientImpl.convertHiveTableToCatalogTable` will affect other opt that use `CatalogTable`, like `desc formatted table`.

Reply:
Other hand, it's no matter add some unexpected properties into new table properties.